tests/includes/juju.sh

```diff
@@ -46,6 +46,9 @@ bootstrap() {
         "aws")
             provider="aws"
             ;;
+        "ec2")
```

Comment:
Why do we need this bit?

Reply:
When running the tests with a already bootstrapped aws controller, the type comes up as ec2.